### PR TITLE
feat: v2 api add GET /players/{player_id}/status

### PR DESCRIPTION
**Pull request by alowave223**

### CODE REVIEW

File: app/api/v2/models/players.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 from __future__ import annotations
 
+from datetime import datetime
+from typing import Literal
```
Comment (cmyui) [inline app/api/v2/models/players.py:4]:
```suggestion
```

File: app/api/v2/players.py [hunk 2]
```diff
@@ -68,3 +71,26 @@ async def get_player(player_id: int) -> Success[Player]:
 
     response = Player.from_mapping(data)
     return responses.success(response)
+
+
+@router.get("/players/{player_id}/status")
+async def get_player(
+    player_id: int,
+) -> Success[PlayerStatus]:
```
Comment (cmyui) [inline app/api/v2/players.py:79]:
```suggestion
async def get_player(player_id: int-> Success[PlayerStatus]:
```

File: app/api/v2/players.py [hunk 3]
```diff
@@ -2,14 +2,17 @@
 from __future__ import annotations
 
 from typing import Optional
+from typing import Union
```
Comment (cmyui) [inline app/api/v2/players.py:5]:
```suggestion
```

**Comment by cmyui** [comment]
LGTM, i'll test & merge this tomorrow

File: app/api/v2/players.py [hunk 4]
```diff
@@ -68,3 +70,24 @@ async def get_player(player_id: int) -> Success[Player]:
 
     response = Player.from_mapping(data)
     return responses.success(response)
+
+
+@router.get("/players/{player_id}/status")
+async def get_player(player_id: int-> Success[PlayerStatus]:
```
[the remark this answers is not in the document]
Comment (alowave223) [reply app/api/v2/players.py]: I think you forgot the closing brackets bro

File: app/api/v2/models/players.py [hunk 5]
```diff
@@ -1,5 +1,6 @@
 from __future__ import annotations
 
+from datetime import datetime
```
Comment (cmyui) [inline app/api/v2/models/players.py:3]:
```suggestion
```

**Review by cmyui — APPROVED** [review]

**Comment by cmyui** [comment]
+1 👍🏻

**Pull request merged**